internal/scorecard/alpha/config.go

```diff
@@ -14,16 +14,43 @@
 
 package alpha
 
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+)
+
 type ScorecardTest struct {
 	Name        string            `yaml:"name"`                 // The container test name
 	Image       string            `yaml:"image"`                // The container image name
 	Entrypoint  string            `yaml:"entrypoint,omitempty"` // An optional entrypoint passed to the test image
 	Labels      map[string]string `yaml:"labels"`               // User defined labels used to filter tests
 	Description string            `yaml:"description"`          // User readable test description
+	TestPod     *v1.Pod           `yaml:"-"`                    // Pod that ran the test
```

Comment:
Will the TestPod filled manually by the user or not? If yes, would not be better it be the NamespacedName .. I mean .. the name and namespace of the pod ressurce?

Reply:
the TestPod there is a means to relate the test with the test pod essentially so when the output is ready to be produced you have both the test and the pod to use for output.